images/router/egress/squid.conf

```diff
@@ -0,0 +1,9 @@
+# egress-router squid.conf
+# This is a mostly-transparent HTTP/HTTPS proxy; it only exists to
+# redirect requests, not to authenticate them in any way.
+
+http_port 8080
```

Comment:
8080 seemed like the most "generic HTTP proxy" port to me? I didn't want to use 3128 because that's explicitly squid's port